From 1d5a594d7fca77155d3dc31044beea1360e7b824 Mon Sep 17 00:00:00 2001
From: Eohan George <eohangeorge@gmail.com>
Date: Fri, 30 Sep 2011 17:49:11 -0400
Subject: [PATCH] changes made to compile wihout file system on Windows 7

---
 flight/PX2FMU/Makefile |   10 ++++++----
 1 files changed, 6 insertions(+), 4 deletions(-)

diff --git a/flight/PX2FMU/Makefile b/flight/PX2FMU/Makefile
index 27ccfd4..2e1af67 100644
--- a/flight/PX2FMU/Makefile
+++ b/flight/PX2FMU/Makefile
@@ -232,8 +232,9 @@ SRC += $(PIOSCOMMONLIB)/msheap/msheap.c
 SRC += $(PIOSCOMMONLIB)/msheap/pios_msheap.c
 
 ## Dosfs file system
-SRC += $(DOSFSDIR)/dosfs.c
-SRC += $(DOSFSDIR)/dfs_sdcard.c
+## The File system is not enabled and the flight\PiOS\STM32F2xx directory has a wierd linking to the flight\PiOS\STM32F2xx directory . 
+#SRC += $(DOSFSDIR)/dosfs.c
+#SRC += $(DOSFSDIR)/dfs_sdcard.c
 
 # List C source files here which must be compiled in ARM-Mode (no -mthumb).
 # use file-extension c for "c-only"-files
@@ -283,8 +284,9 @@ EXTRAINCDIRS  += $(PIOSBOARDS)
 EXTRAINCDIRS  += $(STMSPDINCDIR)
 EXTRAINCDIRS  += $(STMUSBINCDIR)
 EXTRAINCDIRS  += $(CMSISDIR)
-EXTRAINCDIRS  += $(DOSFSDIR)
-EXTRAINCDIRS  += $(MSDDIR)
+## The File system is not enabled and the flight\PiOS\STM32F2xx directory has a wierd linking to the flight\PiOS\STM32F2xx directory . 
+#EXTRAINCDIRS  += $(DOSFSDIR)
+#EXTRAINCDIRS  += $(MSDDIR)
 EXTRAINCDIRS  += $(RTOSINCDIR)
 EXTRAINCDIRS  += $(APPLIBDIR)
 EXTRAINCDIRS  += $(RTOSSRCDIR)/portable/GCC/ARM_CM3
-- 
1.7.5.GIT

